tectonic/msvc-include/unistd.h

```diff
@@ -0,0 +1,2 @@
+// For now, this is a total stub.
+// See https://stackoverflow.com/q/341817 for some things we could want.
```

Comment:
I know essentially nothing about Windows development — is stubbing headers like this a common practice? Feels gross, but you do what you gotta do ...